report_pdf_form/models/ir_actions_report.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2025 Camptocamp SA
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl)
+import base64
+import io
+
+from pytz import timezone
+
+from odoo import api, models
+from odoo.exceptions import UserError
+from odoo.tools import format_amount, format_date, format_datetime, pdf
+from odoo.tools.float_utils import float_compare
+from odoo.tools.pdf import (
+    NameObject,
+    NumberObject,
+    PdfFileReader,
+    PdfFileWriter,
+    createStringObject,
+)
+from odoo.tools.safe_eval import datetime, dateutil, safe_eval, time
+
+
+class IrActionsReport(models.Model):
+    _inherit = "ir.actions.report"
+
+    def _render_qweb_pdf_prepare_streams(self, report_ref, data, res_ids=None):
+        pdf_form_report = self.env["report.pdf.form"].search(
+            [("report_name", "=", report_ref)]
```

Comment:
I think it should search for `report_id` instead of `report_name`.
```suggestion
            [("report_id", "=", report_ref.id)]
```